spec/bolt/transport/ssh_spec.rb

```diff
@@ -142,6 +142,18 @@ def result_value(stdout = nil, stderr = nil, exit_code = 0)
         expect(Time.now - exec_time).to be > 2
       end
     end
+
+    it "doesn't read system config if load_config is specified" do
+      no_user_conf = mk_config(user: nil, password: password, 'host-key-check' => false)
+      allow(Net::SSH::Config)
```

Comment:
Why is this part of the test?

Reply:
Should this be `expect`, asserting that we do use `Net::SSH::Config` by default? Then another test that sets up an executor that doesn't load it with an `expect(...).never`?